src/components/Card.js

```diff
@@ -11,10 +11,11 @@ const Card = ({ handleScore, id}) => {
     <div onClick={handleClick}>
       <img 
         src={cardImages[id]}
+        className='card'
         style={{
-          width: '200px'
+          
         }}
-        alt='buttholes'
```

Comment:
woooooooooooow

Reply:
hahahahaha

Reply:
i knew i was going to forget to remove that alt